pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -1379,21 +1379,21 @@ private class SetPersistence extends CliCommand {
 
         @Parameter(names = { "-e",
                 "--bookkeeper-ensemble" }, description = "Number of bookies to use for a topic", required = true)
-        private int bookkeeperEnsemble;
+        private int bookkeeperEnsemble = 2;
```

Comment:
Because we already have a default value for this configuration, can we mark those parameters as optional?
And we know it shouldn't be smaller than 0. We can check on the client side with `validateValueWith` to avoid unnecessary requests to the broker.

Reply:
it makes sense